env/io_posix.cc

```diff
@@ -320,6 +322,13 @@ IOStatus PosixSequentialFile::InvalidateCache(size_t offset, size_t length) {
 #endif
 }
 
+IOStatus PosixSequentialFile::GetFileSize(uint64_t& size) const {
+  IOOptions io_opts;
+  IOStatus io_s =
+      FileSystem::Default()->GetFileSize(filename_, io_opts, &size, nullptr);
```

Comment:
cc @anand1976  - do you think it's the best way to GetFileSize (same for win case  [port/win/io_win.cc](https://github.com/facebook/rocksdb/pull/9973/files?show-viewed-files=true&file-filters%5B%5D=#diff-d461daf22c23b03cc251e07a31e89974fae6d1b7424056eeffa21de1e833dd55)). 

It looks right to me but I'm not sure.

Reply:
Resolve for now as this part of code is removed.